src/app/app-routing.module.ts

```diff
@@ -55,7 +55,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
Should we fix the routerLink later to use default config 'corrected'?

Reply:
Had a look and we don't have any relative router links with double dot notation so can update this to `corrected` now. Passes unit and e2e tests, for what that's worth!